db/migrate/20170722185159_seed_sponsors.rb

```diff
@@ -0,0 +1,13 @@
+class SeedSponsors < ActiveRecord::Migration
+  def up
+    return unless Sponsor.count == 0
+    sponsors_array = [
+      [400, "http://lightcastletech.com/", "http://becamp.org/assets/lightcastle_s-03aa32173940d7c7161c7be5bfc2bf7ee01eaeec4c5814198fd1cd64e04e6f0a.png"],
```

Comment:
Probably we don't want to use the fingerprinted path here since it will eventually get removed